### PR TITLE
Added InstallAware

### PR DESCRIPTION
**Pull request by msintle**
InstallAware is now included among the InstallShield and WiX third parties.

# InstallAware

This Pull Request (PR) adds information about the InstallAware third party installer, alongside the existing third parties InstallShield and WiX.

## Summary

InstallAware features one-click inclusion of all Microsoft .NET Framework versions in setups it creates. InstallAware's runtime installation mechanism is fully editable and includes scripts which help avoid common installation pitfalls (for example, for the latest Microsoft .NET 4.7 Framework, the InstallAware runtime automatically updates Windows 7 certificates, installs DirectX DLLs, where these would otherwise break the installation with other third parties like InstallShield or WiX).

## Details

InstallAware's runtime installation mechanism features one-click runtime inclusion into setup projects, as well as partial web deployment, and 64-bit compression (maintaining full extraction compatibility on 32-bit systems) which often reduces runtime sizes to 50% of their already-compressed sizes - resulting in an improved user experience and developer experience at the same time.

### CODE REVIEW

**Comment by mairaw** [comment]
Thanks for your contribution @msintle. I sent a request for feedback internally and I'll respond once I hear back.

**Comment by mairaw** [comment]
Hi @msintle, I've discussed this with the team.  We're happy to add InstallAware to the topic, however we'd have to remove the marketing tone of the changes and instead do one of the two options:
1. Follow the [WIX](https://github.com/dotnet/docs/blob/master/docs/framework/deployment/deployment-guide-for-developers.md) model, where you give a background about the product and link to the company's page for more info
2. Follow the [InstallShield](https://github.com/dotnet/docs/blob/master/docs/framework/deployment/deployment-guide-for-developers.md#installshield-deployment) model, where you have a how-to/step-by-step guidance.

Thanks! 

/cc @unniravindranathan

**Comment by msintle** [comment]
Sounds good. Shall we create a new edit and then submit a new pull request?
Thank you.

**Comment by BillWagner** [comment]
@msintle You can add the new commits on the current branch. When you push the changes to GitHub the PR will be automatically updated with the new changes. There's no need to close this and make a new PR.

**Comment by msintle** [comment]
We have followed the WiX example and committed a new set of changes.

WiX Word Count: 71
WiX Link Count: 3

InstallAware Word Count: 67
InstallAware Link Count: 3

Please let us have your thoughts.

File: docs/framework/deployment/deployment-guide-for-developers.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ For download links, see the section [Redistributable Packages](#redistributable-
```
Comment (mairaw) [inline docs/framework/deployment/deployment-guide-for-developers.md]: on line 4, please update ms.date to reflect we're making important changes

File: docs/framework/deployment/deployment-guide-for-developers.md [hunk 2]
```diff
@@ -137,6 +139,10 @@ If you use ClickOnce, InstallShield, or WiX to deploy your app, you can add a de
 
 8.  In the **Property Pages** dialog box, choose **OK**.
 
+<a name="installaware"></a> 
+### InstallAware deployment
+InstallAware builds Windows app (APPX), Windows Installer (MSI), Native Code (EXE), and App-V (Application Virtualization) packages from a single source. Easily [include any version of the .NET Framework](https://www.installaware.com/one-click-pre-requisite-installer.htm) in your setup, optionally customizing the installation by [editing the default scripts](https://www.installaware.com/msicode.htm) (for example, InstallAware pre-installs certificates on Windows 7, without which the .NET 4.7 setup fails). For more information on InstallAware, see the [InstallAware for Windows Installer](https://www.installaware.com/) website.
```
Comment (mairaw) [inline docs/framework/deployment/deployment-guide-for-developers.md]: remove the for example sentence from the parentheses and put it in a new sentence. Such as:
... in your setup, optionally customizing the installation by [editing the default scripts](https://www.installaware.com/msicode.htm). For example, InstallAware pre-installs certificates on Windows 7, without which the .NET 4.7 setup fails.

Also change .NET 4.7 to .NET Framework 4.7.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks for working on the feedback. I think it's almost good to merge. I've left a few comments to be addressed. 

@unniravindranathan would you like to review this as well? Internal review URL is:
https://review.docs.microsoft.com/en-us/dotnet/framework/deployment/deployment-guide-for-developers?branch=pr-en-us-2823

File: docs/framework/deployment/deployment-guide-for-developers.md (hunk 1 above)
Comment (msintle) [reply docs/framework/deployment/deployment-guide-for-developers.md]: Done.

**Review by msintle — COMMENTED** [review]
All requested changes committed.

**Comment by msintle** [comment]
Thank you for your feedback *@mairaw*. We have committed the changes
requested. Please let us know if this is ready to merge. Thank you.

**Review by mairaw — APPROVED** [review]
Thank you @msintle. This is good to merge. I appreciate your patience. I also got approval offline from one of the PMs. It'll go live in our next content push to production, probably later today.

**Pull request merged**